content/applications/productivity/whatsapp.rst

```diff
@@ -159,7 +158,7 @@ To accept the agreements, enter the Facebook password for security purposes, and
 for Developers` dashboard.
 
 .. note::
-   If the Meta business account is prohibited from advertising, claiming an app won't be allowed. To
+   If the Meta business account is prohibited from advertising, claiming an app will not be allowed. To
    resolve this issue navigate to `<https://business.facebook.com/business>`_ for assistance.
```

Comment:
Break line earlier to avoid going over 100 chars and reword to avoid future-tense
```suggestion
   If the Meta business account is prohibited from advertising, claiming an app is not allowed. To
   resolve this issue navigate to `<https://business.facebook.com/business>`_ for assistance.
```